### DISCUSSION TITLE
posts/git-simple-guide/

### DISCUSSION THREAD

**Discussion by giscus** [vegetaz/vegetaz.github.io#12] [General]
# posts/git-simple-guide/

Những ngày tháng sinh viên, lúc mới chập chững bắt tay vào code những project để báo cáo cuối kì. Bọn tui làm theo nhóm nhiều người, chia nhỏ công việc và vai trò khác nhau cho từng cá nhân.

https://bthanhtung.github.io/posts/git-simple-guide/

**Comment by ngominhtrungcmag** [comment]
Intro ấn tượng đó :))))
